pages/Documents/ConversationalAI/ConversationBuilder/Variables&Slots/slots.md

```diff
@@ -84,30 +88,50 @@ Importantly, if the consumer were to start off more generally with, “I need a
+<img style="width:700px" class="fancyimage" src="img/ConvoBuilder/slots_ex3_4.png">
 
-    <img style="width:800px" class="fancyimage" src="img/ConvoBuilder/variables_and_slots/slot_nluassist.png">
+When you use Assist to assign an entity to a question, it automatically creates a rule for each question. In turn, each rule creates a slot that contains a slot variable (e.g., ITEM) and whose value is the entity value (e.g., @ITEM).
 
-    Once completed, we'll have a list of questions that looks like the following:
+<img style="width:800px" class="fancyimage" src="img/ConvoBuilder/slots_ex3_5.png">
 
-    <img style="width:600px" class="fancyimage" src="img/ConvoBuilder/variables_and_slots/slot_questions.png">
+For the “color” question though, we couldn’t use Assist. That’s because Assist can’t be used to assign global entities to questions; Assist can only be used to assign custom entities to questions. So, in the case of the “color” question, we created its rule manually. The rule looks much like shown above: It creates a slot that contains our slot variable (COLOR) and whose value is the entity value (@COLOR).
 
-    <img style="width:600px" class="fancyimage" src="img/ConvoBuilder/variables_and_slots/slot_questions2.png">
+The dialog finishes with a statement that makes use of all of the entity values:
 
-    <img style="width:600px" class="fancyimage" src="img/ConvoBuilder/variables_and_slots/slot_questions3.png">
+<img style="width:700px" class="fancyimage" src="img/ConvoBuilder/slots_ex3_6.png">
 
-    When you assign an entity to a question, this automatically creates a rule for each question. Each rule creates a slot that contains our slot variable (e.g., `item`) and whose value is the entity value (e.g., `@item`).
+How does this dialog work in practice? Let’s examine the following conversation flow:
 
-    <img style="width:800px" class="fancyimage" src="img/ConvoBuilder/variables_and_slots/slot_fill.png">
+<img style="width:500px" class="fancyimage" src="img/ConvoBuilder/slots_ex3_7.png">
 
-4. Test the bot using an intent with slot choices as part of the query. When you enter the dialog, if a consumer has supplied an entity that is known to the domain, it will automatically populate the slot, skip the interaction, and move on to the next interaction's question. 
+Notice that the consumer’s initial intent includes an entity value for the ITEM entity. As a result, the ITEM slot is filled automatically, and the “item” question is skipped.
 
-<img style="width:400px" class="fancyimage" src="img/ConvoBuilder/variables_and_slots/slot_fill_prompt.png">
+If the consumer’s initial intent fills all of the slots, the flow moves directly to the confirmation statement:
 
-If a consumer expresses all the slots as part of their intent query, it will skip to our confirmation step.
+<img style="width:500px" class="fancyimage" src="img/ConvoBuilder/slots_ex3_8.png">
```

Comment:
@mferreira-lp, if you have bandwidth for this, please add alternative text (via `alt` attribute) to all images.

A heads-up (also FYI @itay1313), we’re going to require a reasonable alternative description for all images in the near future, so as to improve the accessibility of the dev site.

Reply:
Hi @j9t , I don't have bandwidth to do this today, but I will put this on my To Do list of things to update. Thank you for the heads-up on this new requirement.

Reply:
Fair enough, thanks @mferreira-lp!